### DISCUSSION TITLE
How to send intent with multiple extras

### DISCUSSION THREAD

**Discussion by jrm200306** [Waboodoo/HTTP-Shortcuts#319] [Q&A] [answered]
Hi
I want sent variables to KWGT
KWGT need multiple extras to process it
Tasker's send intent function worked
![Screenshot_20221208-100310](https://user-images.githubusercontent.com/31978/206338461-f766557f-718a-41ce-8612-81d35169fac9.png)

But I don't know how to write code in javascript section
Help me plz

**Comment by Waboodoo** [comment, marked as answer]
To achieve this, you can copy-paste the following code snippet into one of the Scripting feature's text areas:

```js
sendIntent({
    type: 'broadcast',
    packageName: 'org.kustom.widget',
    className: 'org.kustom.lib.editor.tasker.EditVarActivity',
    extras: [
        {
            name: 'org.kustom.action.EXT_NAME',
            type: 'string',
            value: 'httpreq',
        },
        {
            name: 'org.kustom.action.VAR_NAME',
            type: 'string',
            value: 'upload',
        },
        {
            name: 'org.kustom.action.VAR_VALUE',
            type: 'int',
            value: 1024,
        },
    ],
});
```

For more information on the `sendIntent` function, see https://http-shortcuts.rmy.ch/scripting#send-intent
For more information on Scripting in general, see https://http-shortcuts.rmy.ch/scripting

**Reply by jrm200306** [reply]
It works perfect!Thank you!
